core/corehttp/gateway_handler.go

```diff
@@ -125,6 +125,10 @@ func (i *gatewayHandler) getOrHeadHandler(w http.ResponseWriter, r *http.Request
 	i.addUserHeaders(w) // ok, _now_ write user's headers.
 	w.Header().Set("X-IPFS-Path", urlPath)
 
+	//robots-header to prevent search-engines from indexing the gateway because of the duplicate-content problematic when makeing the gateway public
+	w.Header().Set("X-Robots-Tag", "noindex")
```

Comment:
Imagine a world where 50 gateways are public accessible like ipfs.io.
Every piece of content would be duplicated 50 times in the search-engines. Search-engines would have to choose the most relevant version which is not possible for them because the content is exact identical. Also gateway-providers would get into an unknown juridical position because of the 'maybe-stolen' content.

Also ipfs-content has no subdomain per owner. This means you would share trust and penalties with people you dont have any relation to. see https://en.wikipedia.org/wiki/Google_penalty

You mentioned in #1751 to make a canonical-tag which declares the content to be owned by https://ipfs.io which I assume is as bad idea because it will destroy a part of the decentralisation ipfs should provide.
If you want your ipfs-content to be crawled, you should create an own ipfs-proxy which only shows content created by yourself to the clearnet.